src/OaiPmh/Harvester.php

```diff
@@ -312,9 +338,13 @@ protected function getRecords($params)
 
         // Save the records from the response:
         if ($response->ListRecords->record) {
+            $newRecords = count($response->ListRecords->record); 
             $this->writeLine(
-                'Processing ' . count($response->ListRecords->record) . " records..."
+                '[sum: ' . $this->recordsCount .'] Processing '
```

Comment:
A minor point, but I'd perhaps format the message more like:

[x records harvested] Processing y records...

@EreMaijala, what do you think?

I'd also probably be satisfied by just changing "sum" to "total" which seems like it might be a slightly better word.